openapi.yaml

```diff
@@ -279,6 +282,40 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/GenericApiResponse"
+  /getSchemaRegEnvs:
+    get:
+      operationId: schemaRegEnvsGet
+      summary: get schema registry environments
+      tags:
+        - SchemaRequest
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Environment"
+  /uploadSchema:
+    post:
+      operationId: uploadSchema
```

Comment:
Question: I notice that we invert entity and verb on `operationId` in regards to the path. like, `/getEnvs` becomes `environmentsGet`. Should it be the case here too? Is this a convention we want to follow? I feel it probably makes it easier on the autocomplete :o

Reply:
Oh, true! I'm not sure if we have a pattern/convention here how `operationId` should be defined 🤔  Maybe it would be good to _have_ one?

Reply:
I think it would be good, yes! Either we copy exactly the path, or we come up with maybe a m,ore eloquent name, like `{verb}{entity}` (which is pretty much what we currently do, I think).

Reply:
@SmuliS  do you have preferences for naming patterns for `operationId`?

Reply:
`{entity}{verb}` is my preference. (but without reading the code I do not recall what pattern I have used previously)

Reply:
As far as I see, we're mostly using `{entity}{verb}` so I would vote going for that. Should I update it for _all_ in my PR or only for the ones I added? 🤔

Reply:
I updated it only for the ones related to my changes, will add a housekeeping ticket!